api/Request.json

```
@@ -1097,7 +1097,7 @@
               "version_added": "15"
             },
             "firefox": {
-              "version_added": false
+              "version_added": "132"
```

Thanks Hamish. Reverting this then

```suggestion
              "version_added": false
```